assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -936,11 +973,29 @@ const baseSelectors = {
 				: [];
 		}
 	),
+
+	/**
+	 * Checks if the given module has access.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @param {string} slug  Module slug.
+	 * @return {(boolean|undefined)} `boolean` if the module has check access. If the state is still being resolved, returns `undefined`.
+	 */
+	hasModuleAccess( state, slug ) {
+		if ( ! state?.moduleAccess ) {
+			return undefined;
+		}
```

Comment:
If we add the default value for the `moduleAccess` to the `baseInitialState` (see my comment above), then we won't need this check.